src/xcode/ENA/ENA/Source/Views/ExposureSubmission/DynamicLegalCell.swift

```diff
@@ -34,10 +34,11 @@ class DynamicLegalCell: UITableViewCell {
 	///   - accessibilityIdentifier: Optional, but highly recommended, accessibility identifier.
 	func configure(title: NSAttributedString, description: NSAttributedString?, bulletPoints: [NSAttributedString]?, accessibilityIdentifier: String? = nil) {
 		// create a fake label to get the default font for these
-		let label = ENALabel()
+		let label = ENALabel(style: .body)
```

Comment:
This change could affect all occurrences of DynamicLegalCells....did you have a look on all other occurrences?

Reply:
Hi, I have checked almost all of them and it looks good.